crates/forge/src/running.rs

```diff
@@ -94,6 +93,42 @@ pub(crate) fn blocking_run_from_test(
         if send.is_closed() {
             return Ok(TestCaseSummary::Skipped {});
         }
+        let run_result = run_test_case(
+            vec![],
+            &case,
+            &runner,
+            &runner_config,
+            &runner_params,
+            &send_shut_down,
+        );
+
+        if send.is_closed() {
+            return Ok(TestCaseSummary::Skipped {});
+        }
```

Comment:
This is duplicated now

Reply:
yes, but I had to add it to fix our --exit-first tests. We can remove it when we will change the way of testing it.